main.py

```diff
@@ -1,12 +1,176 @@
+
+        elif choice=='3':
+            print(f"Great! Let's update the customer info: {customer.selected_customer}")
+            name=input("First and last name? ")
+            phone=input("What is your phone number? ")
+            postal_code =input("Enter your 5 digit zip code!")
+            response = customer.update_customer(name=name, phone=phone, postal_code=postal_code)
+            print(response)
+            # print("Updated customer:", response["customer"])
+
+        elif choice=='4':
+            customer.delete_customer()
+            print("Customer has been deleted.")
+
+            print(customer.list_customers())
+
+        elif choice=='5':
+            print("List of all customers!")
+            for cust in customer.list_all_customers():
+                print(cust)
+
+        elif choice=='6':
+            id = input("Select the customer id!")
+            if id.isnumeric():
+                id = int(id)
+                customer.get_customer(id=id)
+          
+            if customer.selected_customer:
+                print("Selected customer: ", customer.selected_customer)
+
+        elif choice=='7':
+            print("List of all video!")
+            for vid in video.list_all_videos():
+                print(vid)
+
+        elif choice=='8': 
+            print("Great! Let's add a video.")
+            title=input("Movie Name?")
+            release_date=input("What is the release date? ")
+            total_inventory =input("What is the total inventory?")
+            response = video.add_video(title=title, release_date=release_date, total_inventory=total_inventory)
+            print("New Video:", response["video"])
+
+        elif choice=='9':
+            print("Great! Let's update a video.")
+            title=input("Movie Name?")
+            release_date=input("What is the release date? ")
+            total_inventory =input("What is the total inventory?")
+            response = video.update_video(title=title, release_date=release_date, total_inventory=total_inventory)
+            print(response)
+            print("Updated Video:", response["title"])
+
+        elif choice=='10':
+            id = input("Which id would you like to delete?")
+            video.delete_video(id)
+            print("Video has been deleted.")
+
+            print(video.list_all_videos())
+
+        
+        elif choice=='11':
+            id = input("Select a video id!")
+            if id.isnumeric():
+                id = int(id)
+                video.get_video(id=id)
+          
+            if video.selected_video:
+                print("Selected video: ", video.selected_video)
 
+        elif choice=="12":
+            print("Woot, Let's get you a video or 5!")
+            id= input("What is your customer id?")
+            movie_id= input("Select which movie ids you would like to check out!")
+            rental=RentalOps()
+            rental.check_out(id, movie_id)
+            
```

Comment:
Adding some type of confirmation of a completed checkout process would be super helpful here!